doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con-timer.adoc

```diff
@@ -0,0 +1,67 @@
+[id='con-timer_{context}']
+= Timers
+
+You can use timers to trigger a logic after a certain period or to repeat specific actions at regular intervals. Timers wait for a predefined amount of time before triggering once or repeatedly.
```

Comment:
Suggestion - remove article and clarify logic:

Is there a better word for logic in this case?

s/You can use timers to trigger a logic after a certain period or to repeat specific actions at regular intervals./You can use timers to trigger logic after a certain period or to repeat specific actions at regular intervals.

Reply:
Done